src/commands/dev/dev.ts

```diff
@@ -26,7 +29,9 @@ export default async function dev(
   const cwd = path.join(process.cwd(), dir);
   const port = opts['-p'] || opts['--port'];
   const debug = opts['-d'] || opts['--debug'];
-  const devServer = new DevServer(cwd, { output, debug });
+  const cert = opts['-c'] || opts['--cert'];
+  const key = opts['-k'] || opts['--key'];
```

Comment:
Let's avoid using short hand since we may wish to utilize `-c` or `-k` for something else in the future.

Reply:
I agree. Other options for passing this information:
* Putting this paths as configuration in `now.json`
* Env variables
* Maybe more specific --ssl and we ditch `-c` and `-k` that I added above

I have the feeling that we have to close this PR but if not I'll appreciate your advice on this. I like the minimalistic API that `now` has and would love to keep it as it is. Just wondering what's the most feasible approach.